lib/containers/container_images.pm

```diff
@@ -153,6 +155,24 @@ sub test_opensuse_based_image {
     }
 }
 
+sub ensure_container_rpm_updates {
+    my $diff_file   = shift;
+    my $regex2match = qr/^-(?<package>[^\s]+)\s+(\d+\.?\d+?)+-(?P<update_version>\d+\.?\d+?\.\d+\b).*\s+(\d+\.?\d+?)+-(?P<stable_version>\d+\.?\d+?\.\d+\b)/;
+    my $context     = script_output "cat $diff_file";
+    open(my $data, '<', \$context) or die "problem with $diff_file argument", $!;
+    while (my $line = <$data>) {
+        if ($line =~ $regex2match) {
+            # Use of Dotted-Decimal-Versions. Do not remove 'v' prefix
+            my $updated_v = version->parse("v$+{update_version}");
+            my $stable_v  = version->parse("v$+{stable_version}");
+            record_info("checking... $+{package}", "$+{package} $stable_v to $updated_v");
+            assert_true($updated_v > $stable_v, "$+{package} $stable_v is not updated to $updated_v");
```

Comment:
Wouldn't it be better to use the internals like

    die "$+{package} $stable_v is not updated to $updated_v" unless ($updated_v > $stable_v);

instead of including the `Test:Assert` module here? It's a very minor complain and both should be fine though.

Reply:
i like this. your suggestion is more perlish. Also i have minor changes i want to do. thus i am thinking to merge this and create an additional PR if there is no objection.